testdata/invalid-providerid.json

```diff
@@ -3,8 +3,8 @@
     {
       "retrievalId": "d05a522e-9608-401a-a33f-1eb4ac4111a0",
       "instanceId": "test-instance-id",
-      "cid": "fish",
-      "storageProviderId": "12D3KooWHwRmkj4Jxfo2YnKJC4YBzTNEGDW6Et4E68r7RYVXk46h",
+      "cid": "bafybeic4jpi2detp5n3q6rjo7ckulebtr7dsvt2tbrtcqlnnzqmi3bzz2y",
+      "storageProviderId": "fish",
```

Comment:
should this still be a 12D3... id?

Reply:
it's the invalid one. the test verifies it fails. previously however, the only thing invalid here was the cid, which was not what was being tested.